Gemfile

```diff
@@ -1,4 +1,5 @@
 source "https://rubygems.org"
 
 gem "jekyll"
+gem "jekyll-netlify"
```

Comment:
just curious, what kind of information does this plugin provide?

Reply:
https://rubygems.org/gems/jekyll-netlify
https://github.com/jayvdb/jekyll-netlify
(my first Ruby gem)